library_website/urls.py

```diff
@@ -53,6 +54,7 @@
     ),
     url(r'^about/news-events/events/$', events_view, name='events'),
     url(r'^collex/$', collection_view, name='collection'),
+    url(r'^mailaliases/', mail_aliases_view, name='mail_aliases'),
```

Comment:
I'm still able to access the page at http://wwwdev:8000/mailaliases/. I might be misremembering but I recall that wasn't possible in your original version.

Reply:
When I test this, I am not able to access my page from either wwwdev:8000/mailaliases not from liblet.lib.uchicago.edu/mailaliases